web-common/src/features/alerts/criteria-tab/parseCriteriaError.ts

```diff
@@ -6,5 +6,8 @@ export function parseCriteriaError(errStr: string, index: number): string {
   const match = criteriaParserRegex.exec(errStr);
   if (!match) return "";
   const [, matchedIndex, matchedErr] = match;
-  return Number(matchedIndex) === index ? matchedErr : "";
+  return Number(matchedIndex) === index
+    ? // `value1` in error is not user friendly. replacing with `criteria value`
```

Comment:
Food for thought for later: I was confused myself when I saw "value1". I wonder if there's a way we can restructure the `MeasureFilterEntry` type to make it more self-explanatory.

Reply:
Ya. When we redesign measure filters in dashboard we can get rid of `value1` and `value2`